lib/modules/manager/helm-values/extract.spec.ts

```diff
@@ -47,5 +47,43 @@ describe('modules/manager/helm-values/extract', () => {
       expect(result).toMatchSnapshot();
       expect(result.deps).toHaveLength(5);
     });
+
+    it('extracts from values.yaml image tag"', () => {
+      const input = `
+      replicaCount: 1
+      firstImage:
+        registry: docker.io
+        repository: mjnagel/postgresql12
+        tag: 12.10
+      secondImage:
+        registry: docker.io
+        repository: bitnami/postgres-exporter
+        tag: "10.70"`;
+      const result = extractPackageFile(input);
+      expect(result).toMatchObject({
+        deps: [
```

Comment:
see below and use `theMatchSnapshot`

Reply:
please prefer `toEqual` or `toDeepEqual`. use `toMatchObject` for huge complex objects when only parts are needed to be tested.

avoid `toMatchSnapshot` , only use for huge strings like PR body.

@honkinggoose can you add this to our best practices guide please?

Reply:
> @HonkingGoose can you add this to our best practices guide please?

Will do! PR incoming. 😉